examples/bare-metal/explorer_board/src/app_conf.h

```diff
@@ -8,7 +8,11 @@
 #define appconfAUDIO_CLOCK_FREQUENCY            24576000
 #define appconfPDM_CLOCK_FREQUENCY              3072000
 #define appconfPIPELINE_AUDIO_SAMPLE_RATE       16000
-#define appconfAUDIO_PIPELINE_STAGE_ONE_GAIN    42
+#define appconfINITIAL_GAIN                     (float)20.0
+#define appconfPOWER_THRESHOLD                  (float)0.00001
 #define appconfAUDIO_FRAME_LENGTH            	256
+#define appconfMIC_COUNT                        2
+#define appconfFRAMES_IN_ALL_CHANS              (appconfAUDIO_FRAME_LENGTH * appconfMIC_COUNT)
```

Comment:
The definitions:
MIC_ARRAY_CONFIG_SAMPLES_PER_FRAME
MIC_ARRAY_CONFIG_MIC_COUNT
Are required for the C++ template mic array implementation.  Since these are baked into the mic array IO thread, changing the appconfMIC_COUNT and appconfAUDIO_FRAME_LENGTH in this app as is will not have the expected effect.

Could you replace appconfAUDIO_FRAME_LENGTH with MIC_ARRAY_CONFIG_SAMPLES_PER_FRAME and appconfMIC_COUNT with MIC_ARRAY_CONFIG_MIC_COUNT, as the latter are the actual values.  Alternatively just replace with:
`#define appconfAUDIO_FRAME_LENGTH MIC_ARRAY_CONFIG_SAMPLES_PER_FRAME`
`#define appconfMIC_COUNT MIC_ARRAY_CONFIG_MIC_COUNT`